hanko-js/src/ui/pages/LoginPassword.tsx

```diff
@@ -91,6 +91,8 @@ const LoginPassword = ({ userID, initialError }: Props) => {
           <InputText
             type={"password"}
             name={"password"}
+            autocomplete={"current-password"}
```

Comment:
There is no autocompletion on my end. Tested with Safari and Chrome.

Reply:
It should be the correct autocomplete value for the password input, though. Unfortunately it does not help with password manager limitations with shadow dom.